src/js/core/methods/EthereumSignTypedData.js

```diff
@@ -9,15 +9,17 @@ import type { MessageResponse, EthereumTypedDataStructAck } from '../../types/tr
 import { ERRORS } from '../../constants';
 import type {
     EthereumSignTypedData as EthereumSignTypedDataParams,
-    EthereumSignTypedHash as EthereumSignTypedHashParams,
+    EthereumSignTypedHashOrData as EthereumSignTypedHashOrDataParams,
 } from '../../types/networks/ethereum';
 import { getFieldType, parseArrayType, encodeData } from './helpers/ethereumSignTypedData';
 
-type Params = {
-    address_n: number[],
-    ...$Exact<EthereumSignTypedDataParams>,
-    ...$Exact<EthereumSignTypedHashParams>,
-};
+type Params = $Diff<
+    {
+        address_n: number[],
+        ...$Exact<EthereumSignTypedDataParams> | $Exact<EthereumSignTypedHashOrDataParams>,
+    },
+    { path: any }, // removes the "path" variable from this.params
```

Comment:
The `path` value in `this.params` is never used, and just causes Flowtype issues, so I removed it using [`$Diff<>`](https://flow.org/en/docs/types/utilities/#toc-diff)